msgpack-core/src/main/java/org/msgpack/core/MessagePacker.java

```diff
@@ -32,6 +32,8 @@
 import java.nio.charset.CharsetEncoder;
 import java.nio.charset.CoderResult;
 import java.nio.charset.CodingErrorAction;
+import java.time.Instant;
```

Comment:
drop Java 7 support ?

https://github.com/msgpack/msgpack-java/blob/81d540d55d86862c775f16b9457c586c9127a790/.travis.yml#L12-L13

Reply:
@frsyuki has confirmed it works in Java6 and Java7 even if Instant class is in the import statement. Only programs that uses Instance related methods will fail to compile.